src/components/PeopleList.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect } from 'react';
+import { Loader } from './Loader';
+import { client } from '../utils/fetchPeople';
+import { Person } from '../types';
+import { PeopleTable } from './PeopleTable';
+
+export const PeopleList: React.FC = () => {
+  const [people, setPeople] = React.useState<Person[]>([]);
+  const [getError, setGetError] = React.useState(false);
+  const [isLoading, setIsLoading] = React.useState(false);
+  const womanNames = people.filter((person) => person.sex === 'f');
+  const manNames = people.filter((person) => person.sex === 'm');
+
+  useEffect(() => {
+    const fetchPeople = async () => {
+      setIsLoading(true);
+      try {
+        const response = await client.get<Person[]>('/people.json');
+
+        setPeople(response);
```

Comment:
```suggestion
        const response = await client.get<Person[]>('/people.json');
        
        const preparedPeople = response.map(person => ({
          ...person,
          mother: response.find(
            mother => mother.name === person.motherName,
          ),

          father: response.find(
            father => father.name === person.fatherName,
          ),
        })),

        setPeople(preparedPeople);
```

Reply:
<img width="600" alt="image" src="https://github.com/mate-academy/react_people-table-basics/assets/70220667/00c80bf7-1389-4527-939e-12645bf3e337">
still not fixed